packages/contracts-bedrock/test/properties/medusa/Protocol.properties.t.sol

```diff
@@ -87,7 +100,119 @@ contract ProtocolProperties is ProtocolHandler {
             assert(sourceSupplyBefore - amount == sourceSupplyAfter);
             assert(destinationSupplyBefore + amount == destinationSupplyAfter);
         } catch {
+            // 6
             assert(address(destinationToken) == address(sourceToken) || sourceBalanceBefore < amount);
         }
     }
+
+    /// @custom:property-id 8
+    /// @custom:property calls to sendERC20 with a value of zero dont modify accounting
+    // NOTE: should we keep it? will cause the exact same coverage as property_BridgeSupertoken
```

Comment:
indeed, and there is no bound on the amount in bridgeSupertoken, so it's most likely already tested... Wdyt of keeping it separated but explicitely call `property_BridgeSupertoken(fromIndex, recipientIndex, destinationChainId, 0)` instead (after visibility change)?